packages/storybook/stories/ovhcloud-design-system/accessibility.mdx

```diff
@@ -0,0 +1,383 @@
+
+<Heading label="aria-label: Providing an accessible name" level={ 3 } />
+
+Use <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label">aria-label</ExternalLink> when an element needs a descriptive name but has no visible text.
+
+<Canvas of={ ButtonStories.AccessibilityBadPracticeIconOnly } sourceState="shown" />
+<Icon aria-hidden="true"
+      name={ ICON_NAME.circleXmark }
+      style={{ color: 'var(--ods-color-critical-500)' }} /> Screen readers will announce a button.
+
+<Canvas of={ ButtonStories.AccessibilityIconOnly } sourceState="shown" />
+<Icon aria-hidden="true"
+      name={ ICON_NAME.circleCheck }
+      style={{ color: 'var(--ods-color-success-500)' }} /> Screen readers will announce a button and the aria-label.
+
+<Heading label="Related Sources" level={ 4 } />
+
+* <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label">MDN Documentation - aria-label</ExternalLink>
+* <ExternalLink href="https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#6.1">RGAA - 6.1</ExternalLink>
+* <ExternalLink href="https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.2">RGAA - 7.2</ExternalLink>
+* <ExternalLink href="https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#11.1">RGAA - 11.1</ExternalLink>
+* <ExternalLink href="https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#11.2">RGAA - 11.2</ExternalLink>
+* <ExternalLink href="https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#11.9">RGAA - 11.9</ExternalLink>
+
+<Heading label="aria-labelledby: Referencing another element as a label" level={ 3 } />
+
+Use <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby">aria-labelledby</ExternalLink> to associate an element with an existing label (instead of duplicating text).
+
+<Canvas of={ ButtonStories.AccessibilityBadPracticeLabelledBy } sourceState="shown" />
+<Icon aria-hidden="true"
+      name={ ICON_NAME.circleXmark }
+      style={{ color: 'var(--ods-color-critical-500)' }} /> Screen readers will announce a button.
+
+<Canvas of={ ButtonStories.AccessibilityLabelledBy } sourceState="shown" />
+<Icon aria-hidden="true"
+      name={ ICON_NAME.circleCheck }
+      style={{ color: 'var(--ods-color-success-500)' }} /> When the **Button** is focused, the screen reader should announce the linked label.
+
+<Icon aria-hidden="true"
+      name={ ICON_NAME.circleInfo }
+      style={{ color: 'var(--ods-color-primary-500)' }} /> Use <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label">aria-label</ExternalLink> when there is no visible label, and <ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby">aria-labelledby</ExternalLink> when there is one.
```

Comment:
maybe use `<Message>` here instead as it is not really easy to distinguish from validation example icons above?

Reply:
(same for other info icon message on the whole page)